kratos/python/add_geometries_to_python.cpp

```diff
@@ -226,6 +227,25 @@ void  AddGeometriesToPython(pybind11::module& m)
     PointerVectorPythonInterface<GeometriesArrayType>().CreateInterface(m, "GeometriesVector");
 
     /// Nurbs Geometries
+    // NurbsVolumeGeometry
+    py::class_<NurbsVolumeGeometry<NodeContainerType>, NurbsVolumeGeometry<NodeContainerType>::Pointer, GeometryType >(m, "NurbsVolumeGeometry")
+        .def(py::init<const PointsArrayType&, const SizeType, const SizeType, const SizeType, const Vector&, const Vector&, const Vector&>())
+        .def("PolynomialDegreeU", &NurbsVolumeGeometry<NodeContainerType>::PolynomialDegreeU)
+        .def("PolynomialDegreeV", &NurbsVolumeGeometry<NodeContainerType>::PolynomialDegreeV)
+        .def("PolynomialDegreeW", &NurbsVolumeGeometry<NodeContainerType>::PolynomialDegreeW)
+        .def("KnotsU", &NurbsVolumeGeometry<NodeContainerType>::KnotsU)
+        .def("KnotsV", &NurbsVolumeGeometry<NodeContainerType>::KnotsV)
+        .def("KnotsW", &NurbsVolumeGeometry<NodeContainerType>::KnotsW)
+        .def("NumberOfKnotsU", &NurbsVolumeGeometry<NodeContainerType>::NumberOfKnotsU)
+        .def("NumberOfKnotsV", &NurbsVolumeGeometry<NodeContainerType>::NumberOfKnotsV)
+        .def("NumberOfKnotsW", &NurbsVolumeGeometry<NodeContainerType>::NumberOfKnotsW)
+        .def("IsRational", &NurbsVolumeGeometry<NodeContainerType>::IsRational)
+        .def("NumberOfControlPointsU", &NurbsVolumeGeometry<NodeContainerType>::NumberOfControlPointsU)
+        .def("NumberOfControlPointsV", &NurbsVolumeGeometry<NodeContainerType>::NumberOfControlPointsV)
+        .def("NumberOfControlPointsW", &NurbsVolumeGeometry<NodeContainerType>::NumberOfControlPointsW)
+        .def("GlobalCoordinates", &NurbsVolumeGeometry<NodeContainerType>::GlobalCoordinates)
```

Comment:
We should put this interface directly to the geometry base class

Reply:
Done.